### PR TITLE
feat(token): initialize the client with a token

### PR DESCRIPTION
**Pull request by daddykotex**

### CODE REVIEW

**Comment by jtheoof** [comment]
Don't forget to udpate the `README.md`

**Comment by daddykotex** [comment]
Forgot to document it

File: README.md [hunk 1]
```diff
@@ -50,6 +50,15 @@ var client = new mnubo.Client({
 });
 ```
 
+You can also create a client with an access token. We do not recommend that for production usage:
+```js
+/* Create a new client with client id and client secret. */
```
Comment (jtheoof) [inline README.md]: with pre generated token

File: spec/model/model.spec.js [hunk 2]
```diff
@@ -60,5 +60,18 @@ describe('model', function() {
                 done();
             });
         });
+        it('should work with fixed token', function(done) {
```
Comment (jtheoof) [inline spec/model/model.spec.js]: new line before `it` :)

**Pull request merged**